docs/Projects/TAD/ADRs/0003-tad-tool.md

```diff
@@ -0,0 +1,52 @@
+# ADR-0003 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organizations to get more grip on their algorithms.
+Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a [standardized way](../Reporting-Standard/latest.md).
+
+There are several existing open source
+[tools](../Existing-Tools/Comparison/tools/) which
+we [have assessed](../Existing-Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](../Existing-Tools/Comparison/requirements/) we have.
+
+These are our main requirements:
+
+* The tool offers A one-stop shop for all aspects of the project for all stakeholders for all tasks.
+* The tool supports a workflow to track different stages of the project, it should support lifecycle management.
+* The tool can run many technical tests on a model.
+* The tool supports filling out assessments.
+* The tool supports capturing discussion around technical tests and assessments.
```

Comment:
I would propose to extend this a little bit more. For example, that it supports not only capturing discussions but also collaboration. For example with features like mentions, (email) notifications, status updates, etc. What do you think of this?